etc/docker/ubuntu/README.md

```diff
@@ -11,6 +11,12 @@ Build the main `flux-ubuntu` image:
 $ docker build -f etc/docker/ubuntu/Dockerfile -t ghcr.io/flux-framework/flux-ubuntu .
 ```
 
+If you want to shell inside (and bind for re-compile and development):
+
+```bash
+$ docker run -it -v $PWD:/code ghcr.io/flux-framework/flux-ubuntu
+```
```

Comment:
This is just showing exactly how you can develop the Python stuffs (as I've done in this PR) using the ubuntu container. I think it makes sense to add here because someone would need to know this to develop (at least in a container - I've actually never used/developed flux outside of one!)

Reply:
Yeah that makes sense, I still think it should go in a separate commit. I'm not an authority on the interpretation of RFC 1 though, I'm just interpreting what I've learned from the feedback I got on my own PRs. Will let @grondo lay down the law 😉